client/v2/common/models/boxes_response.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+// BoxesResponse box names of an application
+type BoxesResponse struct {
```

Comment:
This is missing the `next-token` field. I suggest re-running the generator to pick up the latest changes.

Reply:
Fixed in #411